l10n_br_coa/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Please, add readme folder to replace this file using OCA readme autogenerate tools